database/index.jsx

```diff
@@ -0,0 +1,30 @@
+
+const mysql = require('mysql');
+const config = require('../config/config.js');
+
+const db = mysql.createConnection(config);
+
+db.connect((error) => {
+  if (error) {
+    console.log(error);
+  } else {
+    console.log('==mySQL is connected==');
+  }
+});
+
+const getPhotos = (expId, callback) => {
+  const grabAll = `SELECT * FROM photos WHERE experienceId = '${expId}'`;
```

Comment:
You might consider using '?' to prevent XSS attacks. 
`const grabAll = `SELECT * FROM photos WHERE experienceId = ?`;`
`const experienceId = expId`

Reply:
+1. As Dan said in the comments some time ago